test/Dialect/XTenNN/dequantize.mlir

```diff
@@ -0,0 +1,67 @@
+// RUN: aten-opt %s -split-input-file -verify-diagnostics
+
+func.func @valid_dequantize_op_signed(%arg0: tensor<1x2xsi8>) -> tensor<1x2xf32> {
```

Comment:
How to you plan to make this compatible with TOSA in the long run? AFAIK the work to move TOSA to signed types was mentioned but is not planned in the near future.

Reply:
Because we will always cast back to float our tosa ops will be work. The only issue I see is the use of unsigned in the decomposition of these ops. This we need to somehow fix. 

For now when converting the signed case we have to assume `i<> == si<>` ONNX has the same problem.